samples/modules/lvgl/multi_display/prj.conf

```diff
@@ -0,0 +1,22 @@
+CONFIG_MAIN_STACK_SIZE=8192
+CONFIG_LOG=y
+
+CONFIG_DISPLAY=y
+CONFIG_DISPLAY_LOG_LEVEL_INF=y
+
+CONFIG_PRINTK=y
+CONFIG_INPUT=y
+
+CONFIG_LVGL=y
+CONFIG_LV_USE_LOG=y
+CONFIG_LV_LOG_LEVEL_TRACE=y
+CONFIG_LV_Z_MEM_POOL_SIZE=65536
+
+CONFIG_LV_USE_DEMO_WIDGETS=y
+CONFIG_LV_USE_DEMO_FLEX_LAYOUT=y
```

Comment:
Not needed imo, since you are choosing the file in the cmakelists manually anyways. 
Thinking about this it might make sense to move the CMakeLists stuff from `samples/modules/lvgl/demo` to `modules/lvgl/CMakeLists.txt` so this can be pulled in for other samples (like this multidisplay one) more easily.
But as it stands this does nothing atm., so can be removed.